docs/variables.md

```diff
@@ -17,22 +17,25 @@ Examples for valid substitutions: `$(user;1:points)` - get the user with the log
 
 ## Available filters:
 
-`strftime` - allows you to format raw time strings - Help: https://strftime.org\
-`lower` - converts the output of the variable to lowercase\
-`upper` - converts the output of the variable to uppercase\
-`title` - converts the first letter in each word to upper case\
-`capitalize` - converts the first letter of the first word to upper case\
-`swapcase` - swaps all upper case to lower case and vice versa\
-`time_since_minutes` - outputs the time since a certain input time in minutes\
-`time_since` - outputs the time since a certain input time\
-`time_since_dt` - outputs the date since a certain input date\
-`urlencode` - URL encodes the value. see https://en.wikipedia.org/wiki/Percent-encoding\
-`join` - join characters together using `,`\
-`number_format` - format numbers using `,`\
-`or_else` - if the previous filter failed, return what is specified\
-`or_broadcaster`/`or_streamer` - if the previous filter failed, return the streamer's name\
-`slice` - slice the string like a python3 string - see https://www.digitalocean.com/community/tutorials/how-to-index-and-slice-strings-in-python-3 (NOTE: We don't support what they call 'stride')\
-Math - add, subtract, multiply, divide:
+#### Uncategorized
+
+`strftime` - allows you to format raw time strings - Help: https://strftime.org  
+`lower` - converts the output of the variable to lowercase  
+`upper` - converts the output of the variable to uppercase  
+`title` - converts the first letter in each word to upper case  
+`capitalize` - converts the first letter of the first word to upper case  
+`swapcase` - swaps all upper case to lower case and vice versa  
+`time_since_minutes` - outputs the time since a certain input time in minutes  
+`time_since` - outputs the time since a certain input time  
+`time_since_dt` - outputs the date since a certain input date  
```

Comment:
Can we categorize these into a "Time Since" category?

Reply:
we could in another pr yes